TodoApp/SceneDelegate.swift

```diff
@@ -13,7 +13,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     var window: UIWindow?
 
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/coutar-t/Bitrise-CI/blob/542b9b0a748dff3bd6f40f08fe5332fde2be59a7/TodoApp/SceneDelegate.swift#L15"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>SceneDelegate.swift:15</code></td>
    </tr>
  </tbody>
</table>